pkg/cmd/scan_test.go

```diff
@@ -163,103 +164,149 @@ func Test_parseFromFlag(t *testing.T) {
 		},
 		{
 			name: "test invalid",
 			args: args{
-				out: "://",
+				out: []string{"://"},
 			},
-			want: nil,
+			want: []output.OutputConfig{},
 			err:  fmt.Errorf("Unable to parse output flag '://': \nAccepted formats are: console://,html://PATH/TO/FILE.html,json://PATH/TO/FILE.json,plan://PATH/TO/FILE.json"),
 		},
 		{
 			name: "test unsupported",
 			args: args{
-				out: "foobar://",
+				out: []string{"foobar://"},
 			},
-			want: nil,
+			want: []output.OutputConfig{},
 			err:  fmt.Errorf("Unsupported output 'foobar': \nValid formats are: console://,html://PATH/TO/FILE.html,json://PATH/TO/FILE.json,plan://PATH/TO/FILE.json"),
 		},
 		{
 			name: "test empty json",
 			args: args{
-				out: "json://",
+				out: []string{"json://"},
 			},
-			want: nil,
+			want: []output.OutputConfig{},
 			err:  fmt.Errorf("Invalid json output 'json://': \nMust be of kind: json://PATH/TO/FILE.json"),
 		},
 		{
 			name: "test valid console",
 			args: args{
-				out: "console://",
+				out: []string{"console://"},
 			},
-			want: &output.OutputConfig{
-				Key:     "console",
-				Options: map[string]string{},
+			want: []output.OutputConfig{
+				{
+					Key:     "console",
+					Options: map[string]string{},
+				},
 			},
 			err: nil,
 		},
 		{
 			name: "test valid json",
 			args: args{
-				out: "json:///tmp/foobar.json",
+				out: []string{"json:///tmp/foobar.json"},
 			},
-			want: &output.OutputConfig{
-				Key: "json",
-				Options: map[string]string{
-					"path": "/tmp/foobar.json",
+			want: []output.OutputConfig{
+				{
+					Key: "json",
+					Options: map[string]string{
+						"path": "/tmp/foobar.json",
+					},
 				},
 			},
 			err: nil,
 		},
 		{
 			name: "test empty jsonplan",
 			args: args{
-				out: "plan://",
+				out: []string{"plan://"},
 			},
-			want: nil,
+			want: []output.OutputConfig{},
 			err:  fmt.Errorf("Invalid plan output 'plan://': \nMust be of kind: plan://PATH/TO/FILE.json"),
 		},
 		{
 			name: "test valid jsonplan",
 			args: args{
-				out: "plan:///tmp/foobar.json",
+				out: []string{"plan:///tmp/foobar.json"},
 			},
-			want: &output.OutputConfig{
-				Key: "plan",
-				Options: map[string]string{
-					"path": "/tmp/foobar.json",
+			want: []output.OutputConfig{
+				{
+					Key: "plan",
+					Options: map[string]string{
+						"path": "/tmp/foobar.json",
+					},
 				},
 			},
 			err: nil,
 		},
+		{
+			name: "test multiple output values",
+			args: args{
+				out: []string{"console:///dev/stdout", "json://result.json"},
+			},
+			want: []output.OutputConfig{
+				{
+					Key:     "console",
+					Options: map[string]string{},
+				},
+				{
+					Key: "json",
+					Options: map[string]string{
+						"path": "result.json",
+					},
+				},
+			},
+			err: nil,
+		},
+		{
+			name: "test multiple output values with invalid value",
```

Comment:
It could be nice to add test case with two valid outputs to ensure they are not overriden `out: []string{"json:///test.json", "json://test2.json"},`. This does not change actual code coverage, but since it is a possible usecase from user point of view it should be covered IMHO